api.c

```diff
@@ -402,10 +403,6 @@ void getTopClients(char *client_message, int *sock)
 			(insetupVarsArray(clients[j].ip) || insetupVarsArray(clients[j].name)))
 			continue;
 
-		// Hidden client, probably due to privacy level. Skip this in the top lists
-		if(strcmp(clients[j].ip, "0.0.0.0") == 0)
-			continue;
```

Comment:
I don't see a call to `getClientString` to replace this code.

Reply:
At the very beginning of this subroutine, we just terminate if the privacy level is high enough that client IPs are hidden. As the privacy level cannot be lowered on-the-fly any longer, we can never get into a situation where the loop would run over any queries with a sufficient privacy level due to this early returning.

Reply:
If FTL is restarted with a lower privacy level, then it could loop over queries which were saved in the database with hidden client IPs. Would we still want to hide those?

Reply:
Hmm, I agree that having the Top Items being dominated by `hidden` resp. `0.0.0.0` doesn't make much sense. I will reimplement the code to skip such entries in the Top Item replies.